troubleshooting.md

```diff
@@ -143,3 +143,23 @@ If you are using a useradd command within a Dockerfile with a large UID/GID, it
 #### Solution
 
 If the entry in the Dockerfile looked like: RUN useradd -u 99999000 -g users newuser then add the `--log-no-init` parameter to change it to: `RUN useradd --log-no-init -u 99999000 -g users newuser`. This option tells useradd to stop creating the lastlog file.
+
+#### Symptom
+
+If you are running podman or buildah on a home directory that is mounte noexec,
+then they will fail. something like
```

Comment:
Maybe "with a message like" instead of something like?

Reply:
And please cap the start of it.